src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+function main() {
+  const mainFrame = document.querySelector('.gallery__large-img');
+
+  const images = document.querySelectorAll('.list-item__link');
+
+  for (const img of images) {
+    img.addEventListener('click', () => {
+      mainFrame.src = img.href;
```

Comment:
The line `mainFrame.src = img.href;` is likely incorrect. The `img` variable refers to an anchor element, not an image element, so `img.href` will return the URL the anchor points to, not the image source. If the image source is stored in a different attribute, such as `data-src`, you should use `img.getAttribute('data-src')` instead.